universal-application-tool-0.0.1/app/services/applicant/ReadOnlyApplicantProgramServiceImpl.java

```diff
@@ -71,6 +71,36 @@ public String getProgramTitle() {
     return getBlockAfter(block.getId());
   }
 
+  @Override
+  public int getCompletionPercent(String blockId) {
+    ImmutableList<Block> blocks = getAllBlocks();
+
+    double blockCount = blocks.size();
```

Comment:
why `double` instead of `int` here and below?

Reply:
I made it a `double` so that `blockIndex / blockCount` (line 101) returns a `double`. Otherwise I'd have:

`(int) ( (double) blockIndex / (double) blockCount * 100 )`